cmd/sso.go

```diff
@@ -25,9 +28,22 @@ var (
 		GroupID: groupManagementAPI,
 		Use:     "sso",
 		Short:   "Manage Single Sign-On (SSO) authentication for projects",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			signal.NotifyContext(cmd.Context(), os.Interrupt)
```

Comment:
also need to set context so that subsequent calls to `cmd.Context()` use the wrapped context

```suggestion
			ctx, _ := signal.NotifyContext(cmd.Context(), os.Interrupt)
			cmd.SetContext(ctx)
```